libs/common/src/platform/ipc/ipc-message.ts

```diff
@@ -0,0 +1,10 @@
+import { OutgoingMessage } from "@bitwarden/sdk-internal";
```

Comment:
Just because this file is imported into a content script I want to be extra careful we aren't embedding the entire SDK. 
```suggestion
import type { OutgoingMessage } from "@bitwarden/sdk-internal";
```